### ISSUE TITLE
`config/autoprefixer.yml` for Rails Engines

### ISSUE THREAD

**Issue by the-teacher** [ai/autoprefixer-rails#118]
Hello, Andrey!
Just an idea for a small improvement.

Rails app may consist of a couple of Rails engines. An each engine may have its own `config` folder. Sometimes some configuration files may be moved to a Rails Engine to avoid having of a lot of configuration files in the main "root" App.

Looks like right now `autoprefixer-rails` gem may recognize only the path `Rails.root/config/autoprefixer.yml`. 

The following line gives an array of all engines of the current Rails App.

```ruby
[Rails.application] + Rails::Engine.subclasses.map(&:instance)
```

Would be great if this gem may recognize `config` folders of all engines for case when some part of a system has a responsibility only for a view part of the project, and all "view" configurations must be placed in this Engine.

Of course `Rails.root/config/autoprefixer.yml` has to have a higher priority, anyway.

Feel free to reject it if it's out of your roadmap.
Thanks!

**Comment by ai** [comment]
Good idea. Please send a PR.

**Comment by ai** [comment]
Done https://github.com/ai/autoprefixer-rails/pull/122

**Comment by ai** [comment]
Released in 7.1.2.1
